.travis.yml

```diff
@@ -28,6 +29,7 @@ matrix:
 # The order of these tests says which are more likely to run first and fail the whole build fast.
 env:
   - TEST_TYPE=objc-ios
+  - TEST_TYPE=objc-tvos
```

Comment:
Please make this the last entry. We now use `fast_finish: true` (see above). If the objc-ios test or any other test fail before the tvOS tests finishes we should cancel the tvOS test (this is because of the capacity crisis on Travis, if there was enough capacity we'd let all tests finish).